src/main/resources/com/github/mylibrelab/ui/icons/close.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><rect id="frame" width="16" height="16" fill="none"/><path fill-rule="evenodd" clip-rule="evenodd" d="M11.144 4.15l.707.707-3.149 3.141 3.15 3.149-.708.707-3.15-3.149-3.14 3.15-.707-.708 3.148-3.142-3.148-3.148.707-.707 3.148 3.148 3.142-3.148z" fill="#6E6E6E"/></svg>
```

Comment:
Because there already is [`cross.svg`](https://github.com/MyLibreLab/MyLibreLab/blob/34965a386de0a53617537aa48b3deb017d4045eb/src/main/resources/com/github/mylibrelab/ui/icons/cross.svg) maybe we should rename either `cross.svg` or `close.svg` to have a better distinction. Maybe rename `cross.svg` to `close_bold.svg`?

Reply:
Agree. I'll fix this when I get back to my computer.

Reply:
I can't believe I've left this hanging for so long. Apologise, life's been a bit hectic. I'll fix this tomorrow!